CriteoPublisherSdk/Sources/Criteo.m

```diff
@@ -84,7 +84,7 @@ - (void)registerCriteoPublisherId:(NSString *)criteoPublisherId
             CRITEO_PUBLISHER_SDK_VERSION, criteoPublisherId, adUnits.count, adUnits);
       }
     } else {
-      CRLogWarn(@"Registration", @"You can only call register method once");
+      CRLogInfo(@"Registration", @"You should only call register method once");
```

Comment:
I'm worried about publishers using mediation. What do you think about adding a note for them: `Please ignore this if your using mediation adapter.`